data/threatbook

```diff
@@ -0,0 +1,3 @@
+threatbook.cn
+x.threatbook.cn
+s.threatbook.cn
```

Comment:
The last two domains are covered by the first domain, please delete those two subdomains.